drop/drop.d.ts

```diff
@@ -27,7 +27,7 @@ declare class Drop {
     public once(event: string, handler: Function, context?: any): void;
     public off(event: string, handler?: Function): void;
 
-    public static createContext(options: Drop.IDropContextOptions): Drop;
+    public static createContext(options: Drop.IDropContextOptions): Drop.IDropConstructor;
 }
```

Comment:
Try this:
```ts
public static createContext(options: Drop.IDropContextOptions): typeof Drop;
```
This will save you from having to define a duplicate definition for the constructor. "typeof Drop" will give you the type of the constructor function rather than the instance, which is I think what you want.